demo/demoScreen.js

```diff
@@ -80,6 +82,26 @@ export default class KeyboardInput extends Component {
     });
   }
 
+  toggleUseSafeArea = () => {
+    const {useSafeArea} = this.state;
+    this.setState({
+      useSafeArea: !useSafeArea,
+    });
+  }
+
+  safeAreaSwitchToggleForIOS = () => {
+    if (Platform.OS !== 'ios') {
```

Comment:
I understand why you did this, and I'm not saying this necessarily needs to change, but two pointes to consider:
1. Android should be tested (if not testable), so we know there is no red screen.
2. You did not check the iOS phone has SafeArea.

Reply:
1. Android tested and not producing any issues - I decided to keep the toggle also in android to reflect the code state and not the user behavior since it's a demo to showcase capabilities.
2. Same for other iPhones, it shows how your future CustomKeyboard will look like with those different `useSafeArea` values.